learning_assistant/data.py

```diff
@@ -0,0 +1,15 @@
+"""
+Data classes for the Learning Assistant application.
+"""
+from attrs import field, frozen, validators
+from opaque_keys.edx.keys import CourseKey
+
+
+@frozen
+class LearningAssistantCourseEnabledData:
```

Comment:
what's the data class used for?

Reply:
I was following our [Django App Patterns OEP](https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0049-django-app-patterns.html#data-py), so I don't expose a modal instance via the API. I need to return something from `set_learning_assistant_enabled`, because the `CourseApp` ABC requires that the `set_enabled` function implementation returns the value of the toggle.

On second look, I could return a boolean from `set_learning_assistant_enabled` and avoid needing the data class entirely. I think I was treating `set_learning_assistant_enabled` as a create/update operation and was thinking about returning the created/update object, which necessitated the data class. For now, this is a simple model/object, so that could be overkill.

If it feels heavy handed/confusing, let me know. I can return a boolean.

Reply:
makes sense to me 👍